docs/modules/data-structures/pages/vector-search-overview.adoc

```diff
@@ -210,7 +210,8 @@ digraph singleStageSearch {
 The number of partitions has a big impact on the performance of the vector collection. The conflicting factors that can impact the selection of an optimal partition count are as follows:
 
 - *data ingestion*: a greater number of partitions results in improved parallelism, up to around the total number of partition threads in the cluster.
-  After this point, more partitions will not significantly improve ingestion speed.
+  After this point, more partitions will not significantly improve ingestion speed. 
+  If there are fewer partitions than number of cores, not all available resources will be utilized during ingestion because updates on given partition are executed by single thread. 
```

Comment:
```suggestion
  If there are fewer partitions than number of cores, not all available resources will be utilized during ingestion because updates on a given partition are executed by single thread. 
```